src/key.js

```diff
@@ -157,6 +160,16 @@ Key.prototype.toPacketlist = function() {
   return packetlist;
 };
 
+/**
+ * Throws error if signature has revocation key sub packet (not supported by OpenPGP.js)
+ */
+Key.prototype.checkRevocationKey = function(signature) {
```

Comment:
This doesn't really seem like it needs to be a public function exported on the prototype, could you make it just a normal private function?